setup.sh

```diff
@@ -9,7 +9,7 @@ if [ "${0}" != "${BASH_SOURCE}" ]; then
   export PATH=$PWD/bin:$PATH
   export LD_LIBRARY_PATH=$PWD/install/lib:$LD_LIBRARY_PATH
   export CMAKE_PREFIX_PATH=$PWD/install:$CMAKE_PREFIX_PATH
-  export ROOT_INCLUDE_PATH=$PWD/install/include:$ROOT_INCLUDE_PATH
+  export ROOT_INCLUDE_PATH=$PWD/install/include/FCCAnalyses:$ROOT_INCLUDE_PATH
```

Comment:
why that?

Reply:
Not clear... today it works without requiring it. I reverted this change